governance/roles.md

```diff
@@ -68,25 +79,27 @@ The general list of activities for TL are:
 TLs are assigned by CNCF Technical Oversight Committee
 (see [CNCF SIG Tech Lead nomination and election process](https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#elections)).
 
-### Role of project leads
+## Role of project leads
 
 Project Leads will lead larger streams of work that require sustained
-effort and coordination.
+effort and coordination.  In some cases, these projects remain ongoing.
 
 Project Leads are nominated and approved by the following process:
   1. Project Lead actively participates in the group, making a proposal or
   volunteering to take on a project that has been prioritized by the group
-  1. A Chair or TL nominates a candidate.
-  1. The nomination is communicated via a pull request annotating the list of members in the [SIG README](/README.md) with a link to the issue tracking the project. The nomination is typically open for a week (but may be shorter with
-  LGTM of at least two chairs).
+  1. A Co-Chair or TL nominates a candidate.
+  1. The nomination is communicated via a pull request annotating the list of members in the [SIG README](/README.md) with a link to the issue tracking the project. The nomination is typically open for a week (but may be shorter with LGTM of at least two chairs).
   1. Members are encouraged to review the tracking issue and work together
   to ensure that the Project Lead is set up for success or suggest alternatives.
 
+Depending on the expected length of the project, Project Leads may be assigned
+as OWNERS of the project to help facilitate and moderate new content to the
```

Comment:
```suggestion
as OWNERS of the project directory in this repository to help facilitate and moderate new content to the
```

not everyone in the SIG is deeply familiar with github and the OWNERS feature is relatively new, so adding a little context